core/dbt/artifacts/resources/base.py

```diff
@@ -60,6 +61,27 @@ def from_contents(cls, contents: str, name="sha256") -> "FileHash":
         checksum = hashlib.new(name, data).hexdigest()
         return cls(name=name, checksum=checksum)
 
+    @classmethod
+    def from_path(cls, path: str, name="sha256") -> "FileHash":
```

Comment:
Non-breaking change in the `dbt/artifacts/` directory, so I will add the `artifact_minor_upgrade` label to this PR